.travis.yml.in

```diff
@@ -109,8 +109,8 @@ script:
           -DMULTI_THREAD=$MULTI_THREAD
           -DSTATIC=$STATIC
           -DLEAN_EXTRA_MAKE_OPTS=$LEAN_EXTRA_MAKE_OPTS
-          ../src
-  - make -j2
+          ../src || exit
+  - make -j2 || exit
   - cpack
```

Comment:
Just a small optimization to waste less Travis CPU time (and make the log easier to read) with testing if compiling didn't work in the first place. I first tried to add a `set -e`, but that makes the build fail on OSX currently: https://github.com/travis-ci/travis-ci/issues/6307